developer/docs/arvo/marks.md

```diff
@@ -7,125 +7,130 @@ title: Marks
 
 # Marks
 
-We've used predefined marks already, but we haven't yet created
-our own marks.  Let's write a sample mark of our own, then chain
-it together with some preexisting ones to have fun with type
-conversions.
+We've used predefined marks already, but we haven't yet created our own marks. 
+Let's write a sample mark of our own, then chain it together with some 
+preexisting ones to have fun with type conversions.
 
-Let's make a small "examples-cord" mark.  "Cord" is a name we use for `@t`
+Let's make a small "cord" mark. "Cord" is a name we use for `@t`
 text, but there's no predefined mark for it.  Let's put the following code in:
-`/mar/examples/cord.hoon`:
+`/examples/mar/cord.hoon`:
 
 ```
-/?  314
-|_  cod/@t
-++  grab
-  |%
-  ++  atom  |=(arg/@ `@t`(scot %ud arg))
-  --
-++  grow
-  |%
-  ++  md  cod
-  --
---
+/?  314                                                 ::  1
+|_  cod=@t                                              ::  2
+++  grab                                                ::  3
+  |%                                                    ::  4
+  ++  atom  |=(arg=@ `@t`(scot %ud arg))                ::  5
+  --                                                    ::  6
+++  grow                                                ::  7
+  |%                                                    ::  8
+  ++  atom  `@`cod                                      ::  9
+  --                                                    ::  10
+--                                                      ::  11
 ```
 
-`/?  314` is the required
-version number, just like in apps.  After that everything's in a
-`|_`, which is a `|%` core, but with input arguments.  In our
+`/?  314` is the required version number, just like in apps. After that 
```

Comment:
I'm actually not sure what the new Hoon+Zuse version number is, but we should check and change this as it's now something below `314`.